.github/workflows/main.yaml

```diff
@@ -14,14 +15,30 @@ jobs:
       - uses: actions/checkout@v3
       - uses: julia-actions/setup-julia@v2
       - uses: julia-actions/cache@v2
+      - name: Set up custom Julia dependencies
+        run: |
+          using Pkg
+          Pkg.activate("quarto"; shared = true)
+          Pkg.add(url = "https://github.com/asinghvi17/QuartoNotebookRunner.jl", rev = "as/execute-dir")
+          Pkg.instantiate()
+          Pkg.activate(".")
+          Pkg.add([
+            PackageSpec(url = "https://github.com/asinghvi17/Rasters.jl", rev = "as/makie_and_cf"),
+            PackageSpec(url = "https://github.com/rafaqz/DimensionalData.jl", rev = "main"),
```

Comment:
```suggestion
	    # TODO: DimensionalData v0.28 is not compatible with Rasters latest version,
            # and has Makie fixes that we need for the book, otherwise plotting errors out.
	    # These can be removed after Rasters.jl's new breaking version.
            PackageSpec(url = "https://github.com/asinghvi17/Rasters.jl", rev = "as/makie_and_cf"),
            PackageSpec(url = "https://github.com/rafaqz/DimensionalData.jl", rev = "main"),